### ISSUE TITLE
samba release only for one/some nextcloud user

### ISSUE THREAD

**Issue by hurrikan77** [nextcloud/nextcloudpi#1047]
**Description**  
Actually it is possible to activate the sambe sharing function for all users which are created within the nextcloud instance. I would like to have only sambe release for selected user/users.

**Example**  
Nextcloud User 1
Nextcloud User 2
Nextcloud User 3

After selecting the navigation link I would like to have an selection field for which user/users I would like to aktive the samba sharing function.

**Comment by theCalcaholic** [comment]
That would be a nice feature, but is hard to implement with the current ncp-app architecture.
Dynamic configuration items are currently not supported (they are read from static text files atm).

So... good idea, but probably too much effort right now I'd expect.

**Comment by hurrikan77** [comment]
If you create for each user an user group "e.g. samba" within nextcloud.
Is there an possibility to add to the loop for creating samba users an request if the user is member of the "samba" User group.
If yes create the samba sharing, if not skip the user.

**Comment by theCalcaholic** [comment]
Not a bad idea. A "filter_by_group" option might work

**Comment by theCalcaholic** [comment]
@hurrikan77 Out of curiosity: What's your use case for that feature? Why do you want to enable samba only for a portion of your users?

**Comment by nachoparker** [comment]
current support for the feature is very basic. All samba clients share the same password...

**Comment by hurrikan77** [comment]
@theCalcaholic 
I have done a little bit of rasperry pi "recycling" during the holidays. 
I set up some Volumio instances to play music in different rooms. Base is an music depot which I host in my nextcloud instance. I created an new user for that and shared the files of this user via samba with each instance. So I could prevent to copy the whole data on each of the pi's.

That is the case why I asked for only one/some user. For all other users ist not nessesary to have samba Access to there files.
